tools/hermes-parser/js/flow-api-translator/src/flowToFlowDef.js

```diff
@@ -977,6 +978,35 @@ function convertClassDeclaration(
   ];
 }
 
+function convertExpressionToIdOrQualifiedTypeId(
```

Comment:
Naming nit. The input type to `t.InterfaceExtends()` is `id`, so let's match "identifier" as a suitable generic term.

https://github.com/facebook/hermes/blob/831bcdc417493c04456d2c1ba2abbb32d4d21ffb/tools/hermes-parser/js/hermes-estree/src/types.js#L1563

```suggestion
function convertExpressionToIdentifier(
```